agcaballero/portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="nav">
+      <ul id="nav-list">
+        <li class="nav-li"><a href="index.html">About</a></li>
+        <li class="nav-li"><a href="gallery.html">Gallery</a></li>
+        <li class="nav-li"><a href="blog.html">Blog</a></li>
+        <li class="nav-li" id="last-elem">AnneMarie Caballero</li>
+      </ul>
+    </div>
+    
+    <div id="content">
+      <div class="header">
+        <h1>Blog</h1>
+      </div>
+
+      <div class="flex-container" id="blog-container">
+        <div id="blog-nav">
+          <ul>
+            <li>Date - Title</li>
+          </ul>
+        </div>
+
+        <div class="blog">
+          <div class="blog-post blog1">
+            <p class="date">6/24/2020</p>
+            <div class="header">
+              <h3>Fun Title!</h3>
+            </div>
+            <p>Paragraph 1. </p>
+            <br>
+            <p>Paragraph 2. </p>
+            <br>
+            <p>Paragraph 3. </p>
+          </div>
+
+          <div class="blog-post blog2">
+            <p class="date">6/24/2020</p>
+            <div class="header">
+              <h3>Fun Title!</h3>
+            </div>
+            <p>Paragraph 1. </p>
+            <br>
+            <p>Paragraph 2. </p>
+          </div>
+
+          <div class="blog-post blog2">
+            <p class="date">6/24/2020</p>
+            <div class="header">
+              <h3>Fun Title!</h3>
+            </div>
+            <p>Paragraph 1. </p>
+          </div>
+
+          <div class="blog-post blog1">
+            <p class="date">6/24/2020</p>
+            <div class="header">
+              <h3>Fun Title!</h3>
+            </div>
+            <p>Paragraph 1. </p>
+            <br>
+            <p>Paragraph 2. </p>
+            <br>
+            <p>Paragraph 3. </p>
+            <br>
+            <p>Paragraph 4. </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Recommend adding a newline at the end of the file to avoid this warning. This also applies to a few other files in this PR.

Reply:
Ok thank you!